packages/ssh-agent/ssh-agent.0.4.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+synopsis: "Ssh-agent protocol parser and serialization implementation"
+maintainer: "Reynir Björnsson <reynir@reynir.dk>"
+dev-repo: "git+https://github.com/reynir/ocaml-ssh-agent.git"
+homepage: "https://github.com/reynir/ocaml-ssh-agent/"
+bug-reports: "https://github.com/reynir/ocaml-ssh-agent/issues/"
+license: "BSD-2-clause"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08.0"}
```
Due to
```
# (cd _build/default && .ppx/dddc94fd80d07bf5980184f4fc0c28b4/ppx.exe --cookie 'library-name="ssh_agent"' -o src/parse.pp.ml --impl src/parse.ml --dump-ast)
# Fatal error: exception File "src/parse.ml", line 197, characters 2-6: let operators are not supported before OCaml 4.08
#     ocamlopt src/.ssh_agent.objs/native/ssh_agent__Serialize.{cmx,o}
```

Reply:
I think that error is from ppxlib. I think the solution is to add `"ppxlib" {< "version-that-upgraded-to-ast4.08"}` to `conflicts:` (whatever that version is)